src/EnergyPlus/GroundTemperatureModeling/SiteBuildingSurfaceGroundTemperatures.cc

```diff
@@ -70,21 +70,11 @@ std::shared_ptr<SiteBuildingSurfaceGroundTemps> SiteBuildingSurfaceGroundTemps::
     // SUBROUTINE INFORMATION:
     //       AUTHOR         Matt Mitchell
     //       DATE WRITTEN   Summer 2015
-    //       MODIFIED       na
-    //       RE-ENGINEERED  na
 
     // PURPOSE OF THIS SUBROUTINE:
     // Reads input and creates instance of Site:GroundTemperature:BuildingSurface object
 
-    // USE STATEMENTS:
-    using namespace GroundTemperatureManager;
-
-    // Locals
     // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-    bool genErrorMessage = false;
-    int NumNums;
-    int NumAlphas;
-    int IOStat;
```

Comment:
[src/EnergyPlus/GroundTemperatureModeling/SiteBuildingSurfaceGroundTemperatures.cc:84]:(style),[variableScope],The scope of the variable 'genErrorMessage' can be reduced.
[src/EnergyPlus/GroundTemperatureModeling/SiteBuildingSurfaceGroundTemperatures.cc:85]:(style),[variableScope],The scope of the variable 'NumNums' can be reduced.
[src/EnergyPlus/GroundTemperatureModeling/SiteBuildingSurfaceGroundTemperatures.cc:86]:(style),[variableScope],The scope of the variable 'NumAlphas' can be reduced.
[src/EnergyPlus/GroundTemperatureModeling/SiteBuildingSurfaceGroundTemperatures.cc:87]:(style),[variableScope],The scope of the variable 'IOStat' can be reduced.